schemas/vertices/gtdb/gtdb_vertices.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "description": "A entry for vertices in the GTDB taxonomy tree.",
+  "required": ["_key", "release", "type", "name"],
+  "optional": ["domain", "phylum", "class", "order", "family", "genus", "species"]
+  "properties": {
+    "_key": {
+      "type": "string",
+      "description": "Taxon type abbreviation plus name",
+      "examples": ["d:bacteria", "s:['salmonella', 'enterica']"]
+    },
+    "release": {
+      "type": "string",
+      "description": "Name of the tsv file.",
+      "examples": ["bac_taxonomy_r83", "bac120_taxonomy_r89"]
+    },
+    "name": {
```

Comment:
Let's just make this "name" field always an array, rather than either a string or an array -- will make queries easier